src/Stoa.ts

```diff
@@ -366,7 +373,7 @@ class Stoa extends WebService
             (async () => {
                 try
                 {
-                    let max_blocks : bigint = height.value - expected_height.value;
+                    let max_blocks : bigint = height.value - expected_height.value + ((block == null) ? 1n : 0n);
```

Comment:
Add 1 because the block is null and must be recovered to height during catch-up.

Reply:
It's better to add this comment in the code instead of in github.